mvpa2/algorithms/hyperalignment.py

```diff
@@ -198,7 +198,7 @@ def train(self, datasets):
             ref_ds = np.argmax(nfeatures)
         else:
             ref_ds = params.ref_ds
-            if ref_ds < 0 and ref_ds >= ndatasets:
+            if ref_ds >= ndatasets:
```

Comment:
Good point. Those two check are done during declaration in Parameter() as EnsureInt() and EnsureRange(min=0)